src/main/webapp/app/exercises/programming/manage/programming-exercise-detail.component.ts

```
@@ -184,69 +186,77 @@ export class ProgrammingExerciseDetailComponent implements OnInit, OnDestroy {
+                                        this.programmingLanguageFeatureService.getProgrammingLanguageFeature(programmingExercise.programmingLanguage)
+                                            .auxiliaryRepositoriesSupported ?? false;
+                                    this.localVCEnabled = profileInfo.activeProfiles.includes(PROFILE_LOCALVC);
+                                    this.localCIEnabled = profileInfo.activeProfiles.includes(PROFILE_LOCALCI);
+                                    this.irisEnabled = profileInfo.activeProfiles.includes(PROFILE_IRIS);
+                                    if (this.irisEnabled) {
+                                        this.irisSettingsSubscription = this.irisSettingsService.getCombinedCourseSettings(this.courseId).subscribe((settings) => {
+                                            this.irisChatEnabled = settings?.irisChatSettings?.enabled ?? false;
+                                        });
+                                    }
+                                }
+                            })),
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)